relay-event-normalization/src/normalize/mod.rs

```diff
@@ -1390,6 +1390,82 @@ mod tests {
         assert_eq!(&second, &third, "idempotency check failed");
     }
 
+    /// Validate full normalization is idempotent.
+    ///
+    /// Both PoPs and processing relays will temporarily run full normalization
+    /// in events, during the rollout of running normalization once in internal
+    /// relays.
+    // TODO(iker): remove this test after the rollout is done.
```

Comment:
We should keep that, normalization should always be idempotent.

Reply:
Default normalization, yes; but I'm not so sure about non-future compatible steps too. The test above this one verifies default normalization is idempotent.